tests/queries/0_stateless/01683_text_log_deadlock.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
-# Tags: deadlock
+# Tags: deadlock, no-tsan
```

Comment:
It seems we have supressed tsan issues here https://github.com/ClickHouse/ClickHouse/pull/59870/files#diff-3cdbc9c2a0c3de2ee8e6fd929808e956ed2b0a27d183a9040a89863da09d272fR333 Why do we tag `no-tsan` in the tests?

Reply:
I remember I could not get this test and 01393_benchmark_secure_port.sh to run locally. I just saw that they consistently failed in tsan builds. The log files from CI were also not helpful ... so I disabled them for now :-( Not great, but I'll check as a follow up.